infrastructure/src/main/kotlin/app/discord/repository/jpa/user/schema/UserEntity.kt

```diff
@@ -0,0 +1,41 @@
+package app.discord.repository.jpa.user.schema
+
+import jakarta.persistence.*
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.TimeZoneStorage
+import org.hibernate.annotations.TimeZoneStorageType
+import java.time.OffsetDateTime
+
+@Entity(name = "Users")
+class UserEntity(
+    @field: Id
+    @field: GeneratedValue(strategy = GenerationType.IDENTITY)
+    val UsersIdx: Long,
```

Comment:
PrimaryKey 이름은 보편적으로 `id` 를 많이 사용하지만, User Index 라는 명칭으로 사용하고 싶다면 `UserIdx`로 변경하는 것이 더 올바른 명칭으로 보입니다.
```suggestion
    @field: Id
    @field: GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long,
```